test/fixtures/loopback_acl_help.txt

```diff
@@ -0,0 +1,17 @@
+Usage:
+ slc loopback:acl [options] 
```

Comment:
Sure agree, I will create a subfolder for it.

And for
> I think you should require('../lib/globalize') here too, unless I am missing something?

I didn't require `lib/globalize` here since this file only needs `SetRootDir` but not `g.f()`(no need to import `g`), but on a second thought require glob file is easier and more consistent lol.